### PR TITLE
First round of housekeeping

### PR DESCRIPTION
**Pull request by denisonbarbosa**
Addressing some linter and CI issues in our codebase and updating .gitignore

### CODE REVIEW

File: internal/brokers/examplebroker.go [hunk 1]
```diff
@@ -318,6 +319,7 @@ func (b *exampleBroker) IsAuthorized(ctx context.Context, sessionID, authenticat
 	return access, infoUser, nil
 }
 
+//nolint:unparam // This function needs to respect the Brokerer interface, even though it can not error out.
 func (b *exampleBroker) handleIsAuthorized(ctx context.Context, sessionInfo sessionInfo, authData map[string]string) (access, infoUser string, err error) {
```
Comment (didrocks) [inline internal/brokers/examplebroker.go:323]: This is a private method, so it has nothing to doe with the interface, no?
It should then be removed and the linting help on the broker.

File: internal/brokers/localbroker.go [hunk 2]
```diff
@@ -5,21 +5,30 @@ import (
 	"errors"
 )
 
+//nolint:unused // Those should never be called
```
Comment (didrocks) [inline internal/brokers/localbroker.go:8]: Maybe with a better description why we need them, right? Like, we still need to associate the object, but with no implementation.

**Review by didrocks — APPROVED** [review]
Excellent, that will help review the other PR! Let’s get that in once you fix the 2 minor comments.

File: internal/brokers/examplebroker.go (hunk 1 above)
Comment (denisonbarbosa) [reply internal/brokers/examplebroker.go:323]: Oops, you're right. I confused the functions.

Comment (didrocks) [reply internal/brokers/examplebroker.go:323]: and you start to understand now why I dislike those kind of function() -> handleFunction() nesting :p

Comment (denisonbarbosa) [reply internal/brokers/examplebroker.go:323]: :clown_face:

Comment (denisonbarbosa) [reply internal/brokers/examplebroker.go:323]: I still think that the error on the signature is important (to match the IsAuthorized method), so I just updated the comment and fixed the return typo.

Comment (didrocks) [reply internal/brokers/examplebroker.go:323]: Ok, I won’t fight over that one as it’s only some example code which will evolve anyway.

**Pull request merged**